### PR TITLE
BCN - David & Danielle

### PR DESCRIPTION
**Pull request by danielleleb**

### CODE REVIEW

**Comment by danielleleb** [comment]
@davidalexteixeira

File: starter-code/iteration-1/main.ts [hunk 1]
```diff
@@ -1,11 +1,28 @@
 // Iteration 1
 // 1. Create an array of strings
+const tasks = [String]
```
Comment (tawebbcn) [inline starter-code/iteration-1/main.ts]: ```let tasks: Array<string> = [];```

File: starter-code/iteration-1/main.ts [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
-
+const addTask = (newTask) => {
```
Comment (tawebbcn) [inline starter-code/iteration-1/main.ts]: ```const addTask = (newTask): number => {```

Comment (tawebbcn) [reply starter-code/iteration-1/main.ts]: Remember that you can add types to the function. This one should only be able to return a number.

File: starter-code/iteration-1/main.ts [hunk 3]
```diff
@@ -1,11 +1,28 @@
...
+tasks.push(newTask)
+console.log(tasks)
+}
 // 3. Create a function to list all tasks, it must show in the console de task.
+const listAllTasks = () => {
```
Comment (tawebbcn) [inline starter-code/iteration-1/main.ts]: ```const listAllTasks = (): void => {```

File: starter-code/iteration-1/main.ts [hunk 4]
```diff
@@ -1,11 +1,28 @@
...
+    console.log(tasks)
+}
 
 // 4. Create a function to delete a task, you must find the task inside the array and delete it.
+const deleteTask = (searchTerm) => {
```
Comment (tawebbcn) [inline starter-code/iteration-1/main.ts]: ```const deleteTask = (searchTerm): number => {```

File: starter-code/iteration-2/main.ts [hunk 5]
```diff
@@ -1,9 +1,37 @@
 // Add the reference to the "TodoInterface"
+interface TodoInterface {
+    list : string[];
+    addTask(newTask);
+    listAllTasks(tasks);
+    deleteTask(task);
+}
```
Comment (tawebbcn) [inline starter-code/iteration-2/main.ts]:
```
interface TodoInterface {
  list: Array<string>;
  addTask(task: string): number;
  listAllTasks(): void;
  deleteTask(task: string): number;
}
```

File: starter-code/iteration-2/main.ts [hunk 6]
```diff
@@ -1,9 +1,37 @@
...
+
+// let TodoInterface = require('todointerface').default
+
 
 // 1. Create a class Todo that implements the Interface created before.
+class Todo implements TodoInterface {
+    constructor(public list: string[]){
```
Comment (tawebbcn) [inline starter-code/iteration-2/main.ts]: Goood!

File: starter-code/iteration-2/main.ts [hunk 7]
```diff
@@ -1,9 +1,37 @@
...
+
+    }
+    addTask(newTask){
```
Comment (tawebbcn) [inline starter-code/iteration-2/main.ts]: same as iteration 2

**Pull request closed without merging**